### PR TITLE
feat: update function and status codes

### PR DESCRIPTION
**Pull request by muneebkq**
add missing function and status code

### CODE REVIEW

File: examples/LexFloatClient.h [hunk 1]
```diff
@@ -220,6 +294,31 @@ LEXFLOATCLIENT_API int LF_CC GetHostLicenseExpiryDate(uint32_t *expiryDate);
 */
 LEXFLOATCLIENT_API int LF_CC GetFloatingClientMeterAttributeUses(CSTRTYPE name, uint32_t *uses);
 
+/*
+    FUNCTION: GetFloatinglicenseMode()
+
```
Comment (ahmad-kemsan) [inline examples/LexFloatClient.h:299]: FUNCTION: GetFloatingLicenseMode()

**Review by ahmad-kemsan — CHANGES_REQUESTED** [review]
Please updated the function docs.

File: examples/LexFloatClient.h (hunk 1 above)
Comment (muneebkq) [reply examples/LexFloatClient.h:299]: refactored

**Review by ahmad-kemsan — APPROVED** [review]

**Comment by ahmad-kemsan** [comment]
@adnan-kamili I was unable to merge this to the main. The PR has been approved.

**Review by adnan-kamili — APPROVED** [review]

**Pull request merged**